common/changes/@itwin/tree-widget-react/Update_tree_widget_deps_2023-03-23-11-26.json

```diff
@@ -0,0 +1,15 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/tree-widget-react",
+      "comment": "Updated presentation, appui, itwinjs-core, itwinui dependencies and removed SpatialTree",
+      "type": "minor"
+    },
+    {
+      "packageName": "@itwin/tree-widget-react",
+      "comment": "Use itwinui components instead of custom components and itwin variables instead of @itwin/core-react variables",
```

Comment:
```suggestion
      "comment": "Use iTwin UI components instead of custom ones and CSS variables from `@itwin/itwinui-variables` instead of `@itwin/core-react`",
```